docs/rest-api/rest-notebook.md

```diff
@@ -75,8 +75,8 @@ limitations under the License.
     </tr>
     <tr>
       <td>Description</td>
-      <td>This ```POST``` method create a new notebook using the given name or default name if none given.
-          The body field of the returned JSON contain the new notebook id.
+      <td>This ```POST``` methods create a new notebook using the given name or default name if none given.
```

Comment:
Hi @HeartSaVioR : ) 
I guess **This ```POST``` method creates** is correct, not **This ```POST``` methods create**.
Isn't it ? 
If I am right, plz check line number 132, 163.